src/Zap.sol

```diff
@@ -744,6 +745,8 @@ contract Zap is Reentrancy, Errors {
         } catch Error(string memory reason) {
             revert SwapFailed(reason);
         }
+
+        IERC20(_from).approve(ROUTER, 0);
```

Comment:
will you please write a test for this, making sure the allowance is set to zero? should be quite easy to do!

Reply:
oops merged this but will include the test in a test pr today sorry for that.